src/sonic-yang-models/yang-models/sonic-vrrp.yang

```diff
@@ -0,0 +1,111 @@
+module sonic-vrrp {
+
+    namespace "http://github.com/Azure/sonic-vrrp";
+    prefix vrrp;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description "VRRP yang Module for SONiC OS";
+
+    revision 2023-08-10 {
+        description "add sonic-vrrp.yang";
+    }
+
+    container sonic-vrrp {
+
+        container VRRP {
+
+            description "VRRP part of configdb.json"
+
+            list VRRP_LIST {
+
+                key "interface_name vrid";
+
+                leaf interface_name {
+                    type union {
```

Comment:
Please leafref to sonic-interface, sonic-port-channel, sonic-vlan

Reply:
ok.